packages/api/src/authentication/authentication.controller.ts

```diff
@@ -20,10 +21,14 @@ export class AuthenticationController {
         return await this.authService.create(value);
     }
 
-    // To Do: re-write in sign in pr
-    @UseGuards(AuthGuard("local"))
     @Post("signin")
-    async signIn(@Req() req): Promise<User> {
-        return req.user;
+    @UseGuards(AuthGuard("local"))
+    async signIn(@Body() user: UserCredential): Promise<Token> {
+        const { value, error } = signInSchema.validate(user);
+
+        if (error) {
+            throw new BadRequestException();
+        }
+        return this.authService.validateUser(user.email, user.password);
```

Comment:
Use the value object not the body object because its the processed value for convention

```suggestion
        return this.authService.validateUser(value.email, value.password);
```